pkgs/by-name/sh/shiru/0001-Remove-Windows-specific-dep.patch

```diff
@@ -0,0 +1,146 @@
+From 57103351d1c10d6aea2432943a12e07b8b88b08a Mon Sep 17 00:00:00 2001
+From: Naomi Roberts <mia@naomieow.xyz>
```

Comment:
Is this patch available elsewhere online? If so, you could use fetchpatch to import it.
I think that it is going to be a burden to maintain this patch at each version update, so maybe it could be handled somehow upstream.